### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by mugunth87** [TimOliver/TOCropViewController#287]
Im adding support to voiceover in my application and the crop area in the library is not selectable when i have the voiceover turned on. All other control at the bottom are selectable but the crop area cannot be focussed.

**Comment by TimOliver** [comment]
Hmm, admittedly I haven't played too much with the accessibility controls? How would this work exactly? We just need to make the crop view focusable? Would this also mean changing the interactions of it as well?

Either way, sounds like a great idea. If anyone's interested, please file a PR!
